roles/openshift_persistent_volumes/templates/persistent-volume.yml.j2

```diff
@@ -7,6 +7,8 @@ items:
   kind: PersistentVolume
   metadata:
     name: "{{ volume.name }}"
+    labels:
+      name: "{{ volume.name }}"
```

Comment:
can we make this more generic instead? i think we should be able to provide any key and value for a label